themes/cp-theme/red-hat-sass/variables/_typography.scss

```diff
@@ -6,44 +6,68 @@
 // Base Typography
```

Comment:
@mwcz - This file shouldn't be in this PR

Reply:
@kylebuch8  Done, it showed up because I had already merged another branch into this one, and it brought along the typography changes.  It's removed now.